File: exercises/exercise2.c
#include <stdio.h>

int main(){
	printf("This is a pseudo force solver. It takes in an array of masses of the particle and an array of the particle accelerations.");

	int mass[50];
	int acc[50];
	int force[10];

	srand(0);

	for (int i = 0; i < 50; i++){
		mass[i] = 10;
		acc[i] = 10;
		force[i] = mass[i]*acc[i];
		if (force[i] == 100)
			printf("Item %i initialised correctly \n", i);
		else
			printf("ERROR \n");}

	} 

File: exercises/exercise2.py
#!/usr/bin/env python3

import numpy as np

print("This is a pseudo force solver. It takes in an array of masses of the particle and an array of the particle accelerations.");

mass = np.empty(50)
acc = np.empty(50)
force = np.empty(10)

for i in range(50):
    mass[i] = 10;
    acc[i] = 10;
    force[i] = mass[i]*acc[i];
    if force[i] == 100:
        print("Item {} initialised correctly".format(i));
    else:
        printf("ERROR \n");
